app/controllers/api_controller.rb

```diff
@@ -102,6 +102,11 @@ def set_access_control_headers
   ID_ATTRS = %w(href id)
 
   #
+  # Requester types for authentication
+  #
+  REQUESTER_TYPES = {"ui" => :ui_token_ttl}
```

Comment:
Are you expecting other requester types?  I'm concerned about the overhead of adding this when you only have one type, if you don't really have solid plans on adding future types.